includes/libs/JavaScriptMinifier.php

```diff
@@ -562,13 +565,21 @@ public static function minify( $s, $statementsOnOwnLine = false, $maxLineLength
 				$out .= ' ';
 				$lineLength++;
 			}
-			
+			if (
+				$type === self::TYPE_LITERAL
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1067)